deployments/ansible/molecule/custom_vars/windows-converge.yml

```diff
@@ -8,9 +8,9 @@
     splunk_api_url: https://fake-splunk-api.com
     splunk_hec_url: https://fake-splunk-hec.com
     splunk_hec_token: fake-hec-token
-    splunk_otel_collector_version: 0.48.0
+    splunk_otel_collector_version: 0.126.0
     splunk_otel_collector_config: '{{ansible_env.ProgramData}}\Splunk\OpenTelemetry Collector\custom_config.yml'
-    splunk_otel_collector_config_source: ./custom_collector_config_deprecated.yml
+    splunk_otel_collector_config_source: ./custom_collector_config.yml
```

Comment:
This was causing a generic failure on Windows, as the deprecated config was invalid. The failure message on Windows was the service timed out starting up.